scoreboard.js

```diff
@@ -1,16 +1,81 @@
-class Scoreboard {
+const Player = require("./player.js")
+
+module.exports = class Scoreboard {
+    constructor() {
+        // players stores each entry as:
+        // {name: "username", value: SCORE_VAL }
+        this._players = [];
+        this._firstPlace = [];
+        // Both players and firstPlace are arrays of Player objects
+    }
+
+    getWinner() {
+        let winnerStr = "";
+        let isTie = (this._firstPlace.length > 1);
+        for (let i = 0; i < this._firstPlace.length; i++) {
+            if (this._firstPlace.length == 1) {
+                winnerStr = this._firstPlace[i].getName() + " ";
```

Comment:
Can't you break out of the loop here

Reply:
In this case it isn't even a tie so you shouldn't need to enter this loop, just return the string outside of the loop.